src/server/pachyderm_test.go

```diff
@@ -9228,15 +9228,20 @@ func TestDebug(t *testing.T) {
 
 	expectedFiles, pipelines := tu.DebugFiles(t, dataRepo)
 
-	for _, p := range pipelines {
+	for i, p := range pipelines {
+		cmdStdin := []string{
+			fmt.Sprintf("cp /pfs/%s/* /pfs/out/", dataRepo),
+			"sleep 3",
+		}
+		if i == 0 {
+			// fail on purpose
```

Comment:
I think we should add more context to the comment so that looking back we know why this is tested and what we 
should expect